python3-qtodotxt.json

```diff
@@ -0,0 +1,127 @@
+        }
+      ],
+      "modules" : [
+        {
+          "name" : "sip",
+          "sources" : [
+            {
+              "type" : "archive",
+              "url" : "https://sourceforge.net/projects/pyqt/files/sip/sip-4.19.13/sip-4.19.13.tar.gz",
+              "sha256" : "e353a7056599bf5fbd5d3ff9842a6ab2ea3cf4e0304a0f925ec5862907c0d15e"
+            },
+            {
+              "type": "script",
+              "commands": [
+                  "python3 configure.py --sip-module PyQt5.sip -b ${FLATPAK_DEST}/bin -d ${FLATPAK_DEST}/lib/python3.5/site-packages -e ${FLATPAK_DEST}/include -v ${FLATPAK_DEST}/share/sip --stubsdir=${FLATPAK_DEST}/lib/python3.5/site-packages"
+              ],
+              "dest-filename": "configure"
+            }
+          ],
+          "cleanup" : [
+            "/bin",
+            "/include"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "six",
+      "buildsystem": "simple",
+      "build-commands": [
+        "python3 setup.py install --prefix=/app"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/benjaminp/six/archive/1.11.0.tar.gz",
+          "sha256": "927dc6fcfccd4e32e1ce161a20bf8cda39d8c9d5f7a845774486907178f69bd4"
+        }
+      ]
+    },
+    {
+      "name": "python-dateutil",
+      "buildsystem": "simple",
+      "build-commands": [
+        "pip3 install --prefix=/app python_dateutil-2.6.1-py2.py3-none-any.whl"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://pypi.python.org/packages/4b/0d/7ed381ab4fe80b8ebf34411d14f253e1cf3e56e2820ffa1d8844b23859a2/python_dateutil-2.6.1-py2.py3-none-any.whl#md5=342c025339de1e7c2138c74983c111d7",
+          "sha256": "95511bae634d69bc7329ba55e646499a842bc4ec342ad54a8cdb65645a0aad3c"
+        }
+      ]
+    },
+    {
+      "name": "python3-qtodotxt",
+      "buildsystem": "simple",
+      "build-commands": [
+          "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} qtodotxt",
+          "install -Dm644 org.qtodotxt.qtodotxt128x128.png /app/share/icons/hicolor/128x128/apps/org.qtodotxt.qtodotxt.png",
+          "install -Dm644 org.qtodotxt.qtodotxt64x64.png /app/share/icons/hicolor/64x64/apps/org.qtodotxt.qtodotxt.png",
+          "install -Dm644 org.qtodotxt.qtodotxt.desktop /app/share/applications/org.qtodotxt.qtodotxt.desktop",
+          "install -Dm644 org.qtodotxt.qtodotxt.appdata.xml /app/share/appdata/org.qtodotxt.qtodotxt.appdata.xml"
+      ],
+      "sources": [
+        {
+          "type" : "file",
+          "path" : "org.qtodotxt.qtodotxt.desktop"
+        },
+        {
+          "type" : "file",
+          "path" : "org.qtodotxt.qtodotxt.appdata.xml"
+        },
+        {
+          "type" : "file",
+          "path" : "org.qtodotxt.qtodotxt64x64.png"
+        },
+        {
+          "type" : "file",
+          "path" : "org.qtodotxt.qtodotxt128x128.png"
```

Comment:
Where these files come from? If there is SVG available, you should create PNG files from it.

Reply:
they come from a ICO file i converted into an PNG file